packages/phoenix-client/src/phoenix/client/resources/spans/__init__.py

```diff
@@ -491,16 +602,20 @@ async def get_span_annotations_dataframe(
     async def get_span_annotations(
         self,
         *,
-        span_ids: Iterable[str],
+        span_ids: Optional[Iterable[str]] = None,
+        spans: Optional[list[Span]] = None,
```

Comment:
Can use `Iterable` to be more general.
```suggestion
        spans: Optional[Iterable[Span]] = None,
```